telephone-line-segments.el

```diff
@@ -30,6 +30,23 @@
 (telephone-line-defsegment telephone-line-process-segment ()
   mode-line-process)
 
+(telephone-line-defsegment* telephone-line-vc-nobackend-segment ()
```

Comment:
What's this one for? (also, should have a function comment explaining this)

Reply:
Adding a comment in the next commit :S

The vc segment has a "Git " prefix, or whatever the backend is. Realistically it will be git 90% of the time...and even when you have several, you probably know which one :)

Reply:
Yeah, I suppose this works